sdks/python/apache_beam/coders/coders.py

```diff
@@ -453,6 +454,40 @@ def value_coder(self):
     return self
 
 
+class ProtoCoder(FastCoder):
+  """A Coder for Google Protocol Buffers.
+
+  It supports both Protocol Buffers syntax versions 2 and 3. However,
+  the runtime version of the python protobuf library must exactly match the
+  version of the protoc compiler what was used to generate the protobuf
```

Comment:
So a minor version change of protobuf means that we need to regenerate the proto2_coder_test_messages_pb2.py file included in this PR?  Note that we've been using the same proto generated messages for a while now (I don't think we've upgraded the relevant files in internal/clients.